catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/BrickValueParameterTest.java

```diff
@@ -301,8 +301,8 @@ public void testEventBricksDefaultValues() {
 		onData(instanceOf(CollisionReceiverBrick.class)).inAdapterView(BrickPrototypeListMatchers
 				.isBrickPrototypeView())
 				.onChildView(withId(R.id.brick_collision_receive_spinner))
-				.onChildView(withText(CollisionReceiverBrick.ANYTHING_ESCAPE_CHAR + "anything" + CollisionReceiverBrick
-						.ANYTHING_ESCAPE_CHAR))
+				.onChildView(withText(CollisionReceiverBrick.ANYTHING_ESCAPE_CHAR.concat(baseActivityTestRule
+						.getActivity().getString(R.string.collision_with_anything)).concat(CollisionReceiverBrick.ANYTHING_ESCAPE_CHAR)))
```

Comment:
please use getResourcesString(int stringId)  from UiTestUtils, and "+" for string concatination since it makes it a bit more readable.